eCourse_backend/eCourse_backend/static/JS/set_random_pw.js

```diff
@@ -0,0 +1,13 @@
+const Chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz1234567890!$?#";
+const len = 15;
+
+function randomPw() {
+    let res = [];
+    for (let i = 0; i < len; i++) {
+        res.push(Chars.charAt(Math.floor(Math.random() * Chars.length)));
+    }
+    return res.join('');
+}
+
+let pwField = document.getElementById("defaultPWField");
+pwField.setAttribute("value", randomPw());
```

Comment:
This is an ok debug solution. If this were a real software, this would not be ideal

Reply:
Thats the best Solution I found, maybe you can provide the content for the form field from the Backend, but I didn't get it to work how I wanted